crates/vercel/src/vercel.rs

```diff
@@ -49,14 +49,16 @@ impl From<Role> for String {
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 #[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq, EnumIter)]
 pub enum Model {
-    #[serde(rename = "v-0")]
+    #[serde(rename = "v0-1.5-sm")]
     #[default]
-    VZero,
-
+    VZeroOnePointFiveSm,
```

Comment:
went ahead and changed models to currently supported one from [doc](https://vercel.com/docs/v0/api). In case this is not expected please feel free to revert the last commit.